### ISSUE TITLE
Cannot use two usbtv devices simultaneously

### ISSUE THREAD

**Issue by newchief** [simon3z/usbtv#1]
I've compiled this driver as a module for Linux 3.10.25 (I've rewrittten Makefile to achieve that) and for single device, it worked fine. When I connected second device, it cannot start simultaneous grabbing as it fails on ioctl VIDIOC_STREAMON.
When two devices are connected, two device files are created in /dev tree:
/dev/video0 and /dev/video1. When I start grab live image from either of them, I cannot start grab from the other, i.e. when I start grab from /dev/video1, I can't start grab from /dev/video0 until grab on /dev/video1 is stopped. Similarly, when I start grab on /dev/video0 first, I can't start grab on /dev/video1 until grab on /dev/video0 is stopped. Problem is, I need to grab live image from both devices simultaneously. On userspace, all I see is failed ioctl. The only trace that something went wrong in dmesg is:

ehci-pci 0000:00:1d.7: iso sched full d5e93200

Do you have any idea how to overcome this problem?

**Comment by simon3z** [comment]
Can you try to check if it happens also with the current code in the kernel? Instead of compiling/using this module just use the one provided by 3.10.25.
I have the feeling that the issue you're experiencing is preexisting. If indeed that's the case please try to contact the author of the video implementation (Lubomir Rintel).
Sadly I don't have two device to test this myself.

**Comment by newchief** [comment]
I can't find it in 3.10.25 and AFAIK, usbtv was introduced in 3.11 kernel. I'll try to boot 3.13.1 later during the day.

**Comment by newchief** [comment]
I booted 3.13.1 and the problem still exists. Then I installed a card with additional USB 2.0 controller to my PC. And this helped - on both kernels 3.13.1 and 3.10.25. Seems like USB 2.0 is not capable to sustain required bandwith. I also tried to connect both usbtv devices to this new controller - no luck, problem occured again. The only valid setup is each usbtv device connected to different controller (one to onboard, one to additional on card).
